examples/gcp/README.md

```diff
@@ -29,24 +29,37 @@ Log in using Google Cloud credentials.
 ![Account Details](./assets/add-details-gke.png)
 
 5. Assign the roles.
+
 >**NOTE**: The roles you need are Compute Admin, Kubernetes Engine Admin, and Service Account User.
 
 ![Assign Roles](./assets/assign-roles.png)
 
 6. Optionally, you can grant access to this service account to specific users.
 
-7. Save your configuration. 
+7. Save your configuration.
 
 8. In the main **Service Account**  view, create and store the service account key for your account.
 
 ![Create key](./assets/create-account-key-gke.png)
 
-
 ### Run the example
 
 1. To provision a new cluster on GCP, go to the `hydroform` directory and run:
 
-```
+```bash
 go run ./examples/gcp/main.go -p {project_name} -c /{path/to/service_account_key.json}
 ```
-2. Go to **Kubernetes Engine** > **Clusters** to see your cluster on the list.
\ No newline at end of file
+
+2. Go to **Kubernetes Engine** > **Clusters** to see your cluster on the list.
+
+3. Export `GOOGLE_APPLICATION_CREDENTIALS` environment variable pointing to the service account key.  
+
+```bash
+export GOOGLE_APPLICATION_CREDENTIALS={path/to/service_account_key.json}
+```
+
+4. Export `KUBECONFIG` environment variable pointing to the kubeconfig file generated by running the example. This will allow access to the cluster.
```

Comment:
```suggestion
4. Export **KUBECONFIG** environment variable pointing to the `kubeconfig` file generated by running the example. This will allow access to the cluster.
```